src/BlockTypes/ProductCategories.php

```diff
@@ -77,7 +79,7 @@ protected function render( $attributes, $content, $block ) {
 			if ( strstr( $content, 'data-is-dropdown="true"' ) ) {
 				$attributes['isDropdown'] = true;
 			}
-			if ( strstr( $content, 'data-is-hierarchical="false"' ) ) {
+			if ( strstr( $content, 'data-i\s-hierarchical="false"' ) ) {
```

Comment:
Is this change intentional?

Reply:
Hmm no, dunno how that happened must have been during some auto-linting fixes maybe? 🤔